esp-hal/src/timer/systimer.rs

```diff
@@ -7,74 +7,14 @@
 //! timer which can be used, for example, to generate tick interrupts for an
 //! operating system, or simply as a general-purpose timer.
 //!
-//! ## Configuration
```

Comment:
Did you accidentally delete too much? Some of it still looks relevant.

Reply:
I don't think so, which part do you feel is still relevant?

Reply:
Nvm, you've made more code changes since I wrote this comment.